main.go

```diff
@@ -243,6 +247,18 @@ func visit(url *url.URL) {
 	}
 }
 
+func createBody(body string) (io.ReadCloser, error) {
+	if strings.HasPrefix(body, "@") {
+		filename := body[1:]
+		f, err := os.Open(filename)
+		if err != nil {
+			return nil, fmt.Errorf("failed to open data file %s: %v", filename, err)
+		}
+		return f, nil
+	}
+	return ioutil.NopCloser(strings.NewReader(body)), nil
```

Comment:
Why does this need to be a NopCloser? The previous invocation of 
```go
http.NewRequest(httpMethod, url.String(), strings.NewReader(postBody))
```
worked just fine.